Source/Events.Store.Services.Grpc/EventStoreGrpcService.cs

```diff
@@ -56,7 +56,7 @@ public override async Task<Contracts.CommitAggregateEventsResponse> CommitForAgg
             var commitResult = await _eventStoreService.TryCommitForAggregate(
                 new UncommittedAggregateEvents(
                     eventSourceId,
-                    new Artifact(request.Events.AggregateRootId.ToGuid (), ArtifactGeneration.First),
+                    new Artifact(request.Events.AggregateRootId.ToGuid(), ArtifactGeneration.First),
```

Comment:
Off topic, but I know that we don't do anything with the generation right now, but shouldn't it still be set to the one from the request? Or is this just to stop from any craziness coming from the SDKs?

Reply:
Good point. I don't remember if this was intentional or a fluke to be honest 😆